targets/targets.json

```diff
@@ -2441,7 +2441,7 @@
             "TRNG",
             "FLASH",
             "WATCHDOG"
-        ],        
+        ],
```

Comment:
I got confused initially, as there was another PR as well touching these spaces. I left the comment here.